src/components/WeeklyComponent.tsx

```diff
@@ -0,0 +1,70 @@
+import { useCurrentCity } from "@/contexts/currentCity";
+import { WeatherDay } from "@/types";
+import styles from "./WeeklyComponent.module.scss";
+
+export const WeeklyComponent = () => {
+  const { currentCity } = useCurrentCity();
+
+  const weeklyWeather: WeatherDay[] | undefined = currentCity?.days.slice(0, 7)
+
+  const isCurrentDate = (dateString: string): boolean => {
+    // Create a Date object from the date string
+    const givenDate = new Date(dateString);
+
+    // Get the current date
+    const currentDate = new Date();
+
+    return (
+      givenDate.getUTCFullYear() === currentDate.getUTCFullYear() &&
+      givenDate.getUTCMonth() === currentDate.getUTCMonth() &&
+      givenDate.getUTCDate() === currentDate.getUTCDate()
+    );
+  };
+
+  const getWeekday = (dateString: string): string => {
+    const date = new Date(dateString);
+
+    // Array of weekday names
+    const weekdays = [
+      "Monday",
+      "Tuesday",
+      "Wednesday",
+      "Thursday",
+      "Friday",
+      "Saturday",
+      "Sunday",
+    ];
+
+    const dayIndex = date.getDay();
+
+    console.log(dateString);
+
+    if (isCurrentDate(dateString)) {
+      return "Today";
+    }
+
+    return weekdays[dayIndex];
+  };
+
+  return (
+    <div className={styles.WeeklyComponent}>
+      <h2>Weekly Forecast</h2>
+      <ul className={styles.WeeklyComponentList}>
+        {weeklyWeather ? (
+          dailyWeather.map((day, index) => (
+            <li className={styles.WeeklyComponentItem} key={index}>
+              <p className={styles.WeeklyComponentDay}>{getWeekday(day.datetime)}</p>
```

Comment:
According to Aoi's design, if it's today, the day should be "Today"?
But we can modify it after merging it!
![image](https://github.com/Shingo023/weather-app/assets/143541044/a7c60251-6d44-4224-90b6-9bebae6425be)
![image](https://github.com/Shingo023/weather-app/assets/143541044/22da7dc3-fd6a-4f00-8424-e35b3390b66b)